expr/func_removeemptyseries.go

```diff
@@ -48,19 +48,10 @@ func (s *FuncRemoveEmptySeries) Exec(dataMap DataMap) ([]models.Series, error) {
 			}
 		}
 
-		if xffCheck(notNull, len(serie.Datapoints), s.xFilesFactor) {
+		if pointsXffCheck(serie.Datapoints, s.xFilesFactor) {
```

Comment:
why count all the notnull's and not use it? probably should remove that code

Reply:
Good catch. I'll take this out